configman/config_manager.py

```diff
@@ -45,21 +45,35 @@
 import functools
 import warnings
 
-import configman as cm
-import converters as conv
-import config_exceptions as exc
-import value_sources
-import def_sources
-
 #==============================================================================
 # for convenience define some external symbols here - some client modules may
 # import these symbols from here rather than their origin definition location.
 # PyFlakes may erroneously flag some of these as unused
-from option import Option, Aggregation
-from dotdict import DotDict, DotDictWithAcquisition, iteritems_breadth_first
-from namespace import Namespace
-from required_config import RequiredConfig  # used elsewhere - do not remove
-from config_file_future_proxy import ConfigFileFutureProxy
+from configman.command_line import command_line
+from configman.converters import to_string_converters
+from configman.config_exceptions import NotAnOptionError
+from configman.config_file_future_proxy import ConfigFileFutureProxy
+from configman.def_sources import setup_definitions
+from configman.dotdict import (
+    DotDict,
+    DotDictWithAcquisition,
+    iteritems_breadth_first
+)
+from configman.environment import environment
+from configman.namespace import Namespace
+from configman.option import (
+    Option,
+    Aggregation
+)
+# The following is not used directly in this file, but made available as
+# a type to be imported from this module
+from configman.required_config import RequiredConfig
+from configman.value_sources import (
+    config_filename_from_commandline,
+    wrap,
+    write,
+    file_extension_dispatch,
+)
```

Comment:
I would much prefer 
```
from configman import value_sources
....
def foo():
    value_sources.wrap(...)
```

Names like `wrap`, `write` don't mean much without their context, i.e. where they came from. 
Some are OK, like exceptions with good names or things that you need to access often like `from os import path`. 

Again, mainly a personal style choice so feel free to override/ignore.